Microsoft/WebTemplateStudio
# Removed delete folder function of run-tests

Changes:
- removed the portion of the test that deletes the files because it will get automatically deleted in the build pipeline anyways (we'll end up saving about a few minutes)

How to test:
- run yarn template-tests (you can exist as soon as you see yarn install because the test should work since I only deleted the method & created a folder in generate-test)
